Add standalone Application Security guide (#24644)
DataDog/documentation: content/en/security/application_security/guide/standalone_application_security.md

---

> title: Using standalone Application Security products

Reviewer: ```suggestion
title: Set Up Application Security Products without using APM
```

Reviewer: I think this title is a clearer explanation of what this guide does

disable_toc: false
---

> DataDog [Application Threat Management][1] and [Code Security][2] are built on top of [APM][3]. While recommend using our application security products alongside with APM and adopt DevSecOps practices, some organizations are interested in more incremental adoption. This guide explains how to set up Application Security products without using APM.

Reviewer: ```suggestion
Datadog ASM [Threat Management][1] and [Code Security][2] are built on top of [APM][3]. While Datadog recommends using these security products with APM and adopting DevSecOps practices, you can also use these security products without using APM. This configuration is referred to as Standalone Application Security. This guide explains how to set up Standalone Application Security. 
```

## Prerequisites

> This page assumes you already have:
> 
> * **Datadog Agent Installation:** The Datadog Agent is installed and configured for your application’s operating system or container, cloud, or virtual environment.
> * **Supported Tracing Library:** The Datadog Tracing Library used by your application or service supports application security. For more details, refer to the to the setuo guides for [Appplication Threat Management][4] or [Code Security][5].

Reviewer: ```suggestion
This guide assumes you have the following:

- **Datadog Agent installation:** The Datadog Agent is installed and configured for your application’s operating system, container, cloud, or virtual environment.
- **Supported Tracing Library:** The Datadog Tracing Library used by your application or service supports application security. For more details, see the guides for [Application Threat Management][4] or [Code Security][5].
```

## Compatibility

Standalone Application Security is currently supported for the following tracing library versions:

| Language | Version |
| -------- | ------- |
| .NET     | 3.12.0  |
| Go       | N/A     |
| Java     | 1.47.0  |
| Node.js  | 5.40.0  |
| PHP      | N/A     |
| Python   | 2.19.0  |
| Ruby     | N/A     |

## Setup

> ### Agent setup

Reviewer: ```suggestion
```

> The DataDog Agent should be setup like you would normally do for APM or Application Security.
> 
> However, when setting up the Tracing Library, you should add the `DD_APM_TRACING_ENABLED=false` environment variable to the service that runs the Tracing Library.

Reviewer: ```suggestion
Set up the Datadog Agent using the standard method for APM or Application Security setup, but set up the Tracing Library by adding the `DD_APM_TRACING_ENABLED=false` environment variable to the service that runs the Tracing Library.
```

> This environment variable will reduce the amount of APM data sent to Datadog to a minimum required by Application Security products. It can then be combined with environment variables to enable Application Threat Management or Code Security.

Reviewer: ```suggestion
This environment variable will reduce the amount of APM data sent to Datadog to the minimum required by Application Security products. The environment variable can then be combined with environment variables to enable ASM Threat Management or Code Security.
```

> For Application Threat Management, add the `DD_APM_TRACING_ENABLED=false DD_APPSEC_ENABLED=true` environment variables.

Reviewer: ```suggestion
For ASM Threat Management, add the `DD_APM_TRACING_ENABLED=false DD_APPSEC_ENABLED=true` environment variable.
```

> For Code Security, add the `DD_APM_TRACING_ENABLED=false DD_IAST_ENABLED=true` environment variables.

Reviewer: ```suggestion
For Code Security, add the `DD_APM_TRACING_ENABLED=false DD_IAST_ENABLED=true` environment variable.
```

> ## Further reading
> 
> {{< partial name="whats-next/whats-next.html" >}}

Reviewer: ```suggestion
```
